### PR TITLE
Remove optimizer from a checkpoint

### PR DESCRIPTION
**Pull request by negvet**

### CODE REVIEW

File: beta/examples/tensorflow/common/prepare_checkpoint.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+from beta.examples.tensorflow.common.utils import configure_paths
+from beta.examples.tensorflow.object_detection.models.model_selector import get_predefined_config
+from beta.examples.tensorflow.object_detection.models.model_selector import get_model_builder
+
+
+def get_argument_parser():
+    parser = get_common_argument_parser(precision=False,
+                                        save_checkpoint_freq=False)
+
+    parser.add_argument(
```
Comment (alexsu52) [inline beta/examples/tensorflow/common/prepare_checkpoint.py:31]: It looks like, the mode command line attribute is redundant here.

**Review by alexsu52 — CHANGES_REQUESTED** [review]
Does the object detection example resume the model from the checkpoint without optimizer parameters?

File: beta/examples/tensorflow/common/prepare_checkpoint.py (hunk 1 above)
Comment (negvet) [reply beta/examples/tensorflow/common/prepare_checkpoint.py:31]: That is true, actually, the whole `get_argument_parser` function here is redundant.

**Comment by negvet** [comment]
> Does the object detection example resume the model from the checkpoint without optimizer parameters?

Yes, it does resume

File: beta/examples/tensorflow/common/prepare_checkpoint.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    checkpoint = tf.train.Checkpoint(model=compress_model)
+    load_checkpoint(checkpoint, config.ckpt_path)
+
+    checkpoint_manager = tf.train.CheckpointManager(checkpoint, config.checkpoint_save_dir, max_to_keep=None)
+    save_path = checkpoint_manager.save()
+    logger.info('Saved checkpoint: {}'.format(save_path))
+
+
+def main(argv):
+    parser = get_common_argument_parser()
```
Comment (alexsu52) [inline beta/examples/tensorflow/common/prepare_checkpoint.py:78]: Could you check that the help (prepare_checkpoint.py --help) for the prepare_checkpoint.py script is correct?

Comment (negvet) [reply beta/examples/tensorflow/common/prepare_checkpoint.py]: I created a separate argument parser to make help correct

**Review by alexsu52 — APPROVED** [review]

**Pull request merged**